### PR TITLE
Correction in formatting

### PR DESCRIPTION
**Pull request by Jaswantsinghh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ Requirements
...
 ```
 1. Apache2
-1. mysql-server
-1. python3.7
-1. python3-pip
-1. python-mysqldb
-1. django 3.0
-1. libldap2-dev
-1. libmysqlclient-dev
+2. mysql-server
```
Comment (SurajDadral) [inline README.md]: List can be done in markdown format by removing ``` from up and below it.

File: README.md [hunk 2]
```diff
@@ -44,11 +44,11 @@ Steps for installation of OutcomeBasedCourse
 ```bash
 git clone https://github.com/GreatDevelopers/OutcomeBasedCourse
 ```
-1. Create a database for OutcomeBasedCourse
+2. Create a database for OutcomeBasedCourse
```
Comment (SurajDadral) [inline README.md]: This list can be corrected by adding three spaces before lines like:
   ```bash
   git clone https://github.com/GreatDevelopers/OutcomeBasedCourse
   ```

**Review by SurajDadral — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Jaswantsinghh) [reply README.md]: Ok.

File: README.md (hunk 2 above)
Comment (Jaswantsinghh) [reply README.md]: Ohh nice.

**Pull request closed without merging**